easybuild/easyconfigs/f/fgbio/fgbio-2.2.1.eb

```
@@ -0,0 +1,42 @@
+easyblock = 'CmdCp'
+
+name = 'fgbio'
+version = '2.2.1'
+
```

This should also have the Java version suffix

```suggestion
version = '2.2.1'
versionsuffix = '-Java-%(javaver)s

```

---

filename will need updating to match